### ISSUE TITLE
Unable to load converted EPUB / KEPUB

### ISSUE THREAD

**Issue by klauz1984** [janeczku/calibre-web#2266]
**Describe the bug/problem**
I've update Kobo Forma setting file -> api_endpoint=http://[ip]:8083/kobo/[token] 
I can see on my book menu on Kobo Forma all the EPUB library and i can download them
I've converted via Calibre (I've installed add ons for Kepub) some CBR/CBZ files to KEPUB/EPUB and i can see that at http://[ip]:8083/kobo/[token] /v1/library/sync
When I resync device no new books are available
Same issue with files converted with Kindle Comic Convert  

**To Reproduce**
[same of the description]

**Logfile**
no erro on log

**Expected behavior**
After device update new books must be visible on catalog

**Screenshots**

**Environment (please complete the following information):**
Platform 	Linux 5.11.0-46-generic #51-Ubuntu SMP Thu Jan 6 22:14:29 UTC 2022 x86_64 x86_64
Python 	3.9.5 (default, Nov 18 2021, 16:00:48) [GCC 10.3.0]
Calibre_Web 	0.6.15 Beta
Werkzeug 	2.0.2
Jinja2 	3.0.3
pySqlite 	2.6.0
SQLite 	3.34.1
Babel 	2.9.1
Flask-Babel 	2.0.0
Flask-Login 	0.5.0
Flask-Principal 	0.4.0
backports_abc 	0.5
Flask 	2.0.2
iso-639 	0.4.5
PyPDF3 	1.0.5
pytz 	2021.1
requests 	2.25.1
SQLAlchemy 	1.4.29
tornado 	6.1
Wand 	0.6.7
unidecode 	1.2.0
lxml 	4.6.3
flask-wtf 	1.0.0
greenlet 	1.1.2
httplib2 	0.18.1
pyasn1-modules 	0.2.1
pyasn1 	0.4.8
PyYAML 	5.3.1
six 	1.15.0
scholarly 	1.5.1
jsonschema 	3.2.0
Image Magick 	ImageMagick 6.9.11-60 Q16 x86_64 2021-01-25 https://imagemagick.org
ebook converter 	not installed
unrar 	not installed
kepubify 	not installed

**Additional context**
Add any other context about the problem here. [e.g. access via reverse proxy, database background sync, special database location]

**Comment by klauz1984** [comment]
UPDATE:
After many attempts I have noticed that when I create a new book within the Calibe library with the converted file, it is added to the Kobo, but not when added to an existing book. Is there any way to fix?

**Comment by OzzieIsaacs** [comment]
Yeah this is indeed a problem, I don't know how to solve it (beside create a new book entry on every metadata change). 
Kobo saves all books metadata (including, title, author, ...) on first sync in an internal database (on Kobo device). 
Changes later in Calibre-Web don't show up on Kobo even if they are transmitted

**Comment by OzzieIsaacs** [comment]
Update: I found one problem, which could lead to this behavior. I'll check if I can fix it
